app/main/models/payments.py

```diff
@@ -24,12 +24,13 @@ def __init__(self, user, amount, api_response):
         self.amount = amount
         self.api_response = api_response
 
-        send_mail(user.email, "Thanks from Ambit", """
-We are very grateful to you. Blah Blah Blah.""")
+        async_send_mail(user.email, "Thanks from Ambit", """
```

Comment:
This method has been added and it takes the current_app as an argument. Should be called as https://github.com/thealphadollar/TriNetra-Clone/blob/35f9de734052d0a0de232e13434ef7ca01a05663/app/main/controller/train_controller.py#L32